app/controllers/feature_controller.rb

```diff
@@ -0,0 +1,45 @@
+class FeatureController < ApplicationController
+  before_action :validate_feature!
+
+  # Toggles boolean for session based feature flags
+  def toggle
```

Comment:
Would it be possible to have a way to check the status of any of the feature toggles as a batch and display a message? Something like ``/toggle?feature=report`` - I'm guessing if we have this enabled on staging we'll set something and forget and have to debug the changed behavior at some point.